ts/components/conversation/right-panel/overlay/message-info/components/MessageInfo.tsx

```diff
@@ -57,6 +64,48 @@ const showDebugLog = () => {
   ipcRenderer.send('show-debug-log');
 };
 
+const showDebugMessageInfo = false;
+
+const DebugMessageInfo = ({ messageId }: { messageId: string }) => {
```

Comment:
Should this not be in a separate component file and then imported?

Reply:
I'd rather keep it here. This fine is small enough and I don't want that component be exported in the global namespace for no reason